doc/cephfs/troubleshooting.rst

```diff
@@ -10,6 +10,13 @@ where the problem is occurring: in the client, the MDS, or the network connectin
 them. Start by looking to see if either side has stuck operations
 (:ref:`slow_requests`, below), and narrow it down from there.
 
+We can get hints about what's going on by dumping the MDS cache ::
+
+  ceph daemon mds.<name> dump cache /tmp/dump.txt
+
+If high logging levels are set on the MDS, that will almost certainly hold the
+information we need to diagnose and solve the issue.
```

Comment:
Add note that the file is written to `/tmp` and that for systemd controlled MDS services, this is in a tmpfs in the MDS container. Use nsenter(1) to locate `dump.txt` or specify another system-wide path.

Reply:
Oops! I missed it.

Reply:
Fixed. Please check.